### PR TITLE
build: fix docker warnings

### PR DESCRIPTION
**Pull request by anshbansal**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: docker/kafka-setup/Dockerfile
Comment (pedro93) [inline docker/kafka-setup/Dockerfile:9]: Likely a typo not needed for documentation.

**Review by pedro93 — APPROVED** [review]
LGTM, just a small nitpick. 
Please wait for green CI before merging.

**Review by hsheth2 — APPROVED** [review]

**Pull request merged**
